crates/core/src/mock/mock_starknet.rs

```diff
@@ -177,4 +235,10 @@ mod tests {
         assert_eq!(expected_params, fixture.params);
         assert_eq!(expected_response, fixture.response);
     }
+
+    #[test]
+    fn test_rpc_all_fixtures() {
+        let fixtures = all_fixtures();
+        assert_eq!(fixtures.len(), 22);
```

Comment:
not sure this is worth it -> if someone adds new fixtures, they'll have to update it

Reply:
not sure neither!

Reply:
deleted it